.github/workflows/notify_slack.yml

```
@@ -9,9 +9,10 @@ jobs:
     timeout-minutes: 5
     if: github.event.discussion && github.event.comment
     steps:
-      - uses: slackapi/slack-github-action@v1.26.0
-        with:
-          channel-id: ${{ secrets.SLACK_CHANNEL_ID }}
-          slack-message: "Created discussion comment: ${{ github.event.comment.html_url }}"
+      - uses: slackapi/slack-github-action@v2.0.0
+        method: chat.postMessage
+        payload: |
+          channel: ${{ secrets.SLACK_CHANNEL_ID }}
```

IMO & nits

channel_id が文字列なのはわかっているのですが，
一応，クォートしておいてもいいかな，と思いました．

```suggestion
          channel: "${{ secrets.SLACK_CHANNEL_ID }}"
```